plugins/janus_videoroom.c

```diff
@@ -2487,11 +2490,20 @@ int janus_videoroom_init(janus_callbacks *callback, const char *config_path)
 	{
 		janus_config_category *config_general = janus_config_get_create(config, NULL, janus_config_type_category, "general");
 		/*CARBYNE-RF*/
-		janus_config_item *item_rtsp = janus_config_get(config, config_general, janus_config_type_item, "rtsp_url");
-		if (item_rtsp && item_rtsp->value)
+		janus_config_item *dynamic_url = janus_config_get(config, config_general, janus_config_type_item, "dynamic_url");
+		if (dynamic_url && !janus_is_true(dynamic_url->value) ){
+			janus_config_item *item_rtsp = janus_config_get(config, config_general, janus_config_type_item, "rtsp_url");
+			if (item_rtsp && item_rtsp->value)
+			{
+				JANUS_LOG(LOG_INFO,"RTSP Dynamic URL is disabled using static\n");
+				rtsp_url = g_strdup(item_rtsp->value);
+				JANUS_LOG(LOG_VERB, "rtsp_url: %s\n", rtsp_url);
+			}
+		}
+		else
 		{
-			rtsp_url = g_strdup(item_rtsp->value);
-			JANUS_LOG(LOG_VERB, "rtsp_url: %s\n", rtsp_url);
+			dynamic_url_status = TRUE;
+			JANUS_LOG(LOG_INFO,"RTSP Dynamic URL is enabled\n");
```

Comment:
`JANUS_LOG(LOG_INFO,"RTSP Dynamic URL is enabled\n");` -> `JANUS_LOG(LOG_INFO, "RTSP Dynamic URL is enabled\n");`

Reply:
fixed